src/commands/deploy/failureMoreInfoSurvey.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { WebSiteManagementModels } from '@azure/arm-appservice';
+import { env, MessageItem, Uri, window } from 'vscode';
+import * as appservice from 'vscode-azureappservice';
+import { DialogResponses, IParsedError } from 'vscode-azureextensionui';
+import { localize } from "../../localize";
+
+const SURVEY_URL: string = 'https://aka.ms/AppServiceExtDeploymentFeedback';
+
+function shouldPromptForSurvey(error: IParsedError, siteConfig: WebSiteManagementModels.SiteConfigResource): string | null {
+    if (siteConfig.linuxFxVersion && siteConfig.linuxFxVersion.toLowerCase().startsWith(appservice.LinuxRuntimes.python)) {
```

Comment:
Fyi, I [moved](https://github.com/microsoft/vscode-azureappservice/pull/1824) `LinuxRuntimes` from the `appservice` package to a file in this repo:

https://github.com/microsoft/vscode-azureappservice/blob/main/src/commands/createWebApp/LinuxRuntimes.ts